quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.approved.txt

```diff
@@ -64,11 +64,7 @@ Cache:
 --cache-embedded-users-max-count <max-count>
                      The maximum number of entries that can be stored in-memory by the users cache.
 --cache-remote-host <hostname>
-                     The hostname of the remote server for the remote store configuration. It
-                       replaces the 'host' attribute of 'remote-server' tag of the configuration
-                       specified via XML file (see 'cache-config-file' option.). If the option is
-                       specified, 'cache-remote-username' and 'cache-remote-password' are required
-                       as well and the related configuration in XML file should not be present.
+                     The hostname of the external Infinispan cluster.
```

Comment:
Should we add more details on when is this option used?

Reply:
https://github.com/keycloak/keycloak/issues/34536 will add a message that this option is available when the feature multi-site is enabled.